core/docker/Dockerfile

```
@@ -30,6 +41,7 @@ ARG TRINO_VERSION
 COPY trino-cli-${TRINO_VERSION}-executable.jar /usr/bin/trino
 COPY --chown=trino:trino trino-server-${TRINO_VERSION} /usr/lib/trino
 COPY --chown=trino:trino default/etc /etc/trino
+COPY --chown=trino:trino --from=0 /tmp/jvmkill/libjvmkill.so /usr/lib/trino/bin
```

Thanks for teaching that we can have un-named stages (numbered starting from `0`).

```suggestion
COPY --chown=trino:trino --from=builder /tmp/jvmkill/libjvmkill.so /usr/lib/trino/bin
```

Then change L14 to be `FROM eclipse-temurin:17-jdk AS builder`. More idiomatic and familiar to most people AFAIK.

---

Nice, that's definitely easier to understand. The `--from=0` makes me think it's reading from stdin. I had copied that from some other Dockerfile.